frontends/election-manager/src/components/election_manager.tsx

```diff
@@ -74,12 +74,24 @@ export function ElectionManager(): JSX.Element {
         <Route exact path={routerPaths.root}>
           <UnconfiguredScreen />
         </Route>
-        <Route exact path={routerPaths.advanced}>
-          <AdvancedScreen />
-        </Route>
         <Route exact path={routerPaths.electionDefinition}>
           <UnconfiguredScreen />
         </Route>
+        {!areVvsg2AuthFlowsEnabled() && (
+          <Route exact path={routerPaths.advanced}>
+            <AdvancedScreen />
+          </Route>
+        )}
+        {areVvsg2AuthFlowsEnabled() && (
+          <Route exact path={routerPaths.settings}>
+            <SettingsScreen />
+          </Route>
+        )}
+        {areVvsg2AuthFlowsEnabled() && (
+          <Route exact path={routerPaths.logs}>
+            <LogsScreen />
+          </Route>
+        )}
```

Comment:
Though I was previously including Settings and Logs in the nav when there was no election definition, I wasn't rendering these routes/screens when there was no election definition.

There are other changes that need to be made to this no-election-definition branch, but those will come later in https://github.com/votingworks/vxsuite/issues/1979